en/docs/assets/versions.json

```diff
@@ -1,15 +1,19 @@
 {
-  "6.0.0": "master",
-  "current": "5.11.0",
+  "6.0.x": "master",
```

Comment:
`6.0.x` ---> should this be the same that we used in the build script?
Hope that was handled